### PR TITLE
JENKINS-56773 - Disable Executor plugin by default, and throttle requests for the api for those that have it installed

### PR DESCRIPTION
**Pull request by halkeye**
# Description

See [JENKINS-56773](https://issues.jenkins-ci.org/browse/JENKINS-56773).

# Submitter checklist
- [ ] Link to JIRA ticket in description, if appropriate.
- [ ] Change is code complete and matches issue description
- [ ] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

### CODE REVIEW

File: blueocean-executor-info/src/main/js/ExecutorInfoService.js [hunk 1]
```diff
@@ -1,22 +1,25 @@
 import { observable, action } from 'mobx';
 import { Fetch, UrlConfig, AppConfig, sseConnection } from '@jenkins-cd/blueocean-core-js';
+import throttle from 'lodash.throttle';
 
+
+const FETCH_TIMEOUT = 30 /* seconds */ * 1000 /* milliseconds */;
```
Comment (sophistifunk) [inline blueocean-executor-info/src/main/js/ExecutorInfoService.js]: Can we name this `FETCH_TIMEOUT_MS`?

**Comment by halkeye** [comment]
done

**Comment by sophistifunk** [comment]
🐝

**Comment by sophistifunk** [comment]
LGTM

**Comment by tempora-mutantur** [comment]
Thank you. Can you, please, tell when this is going to be released?

**Comment by halkeye** [comment]
Probably tomorrow, it just got merged today, i'm cleaning up a few infra
stuff to finish testing before releasing

**Comment by tempora-mutantur** [comment]
Hi, @halkeye . As I see, the load is still pretty high after the update. 

image

Could you, please, tell if I can disable the blueocean-executor-info plugin because the dependency was disabled?
Thank you

**Comment by halkeye** [comment]
yea go ahead and disable it, its an optional addon

**Pull request merged**